docs/src/modules/components/MarkdownDocs.js

```diff
@@ -38,10 +41,39 @@ const styles = theme => ({
     marginBottom: theme.spacing(2),
     padding: theme.spacing(0, 1),
   },
+  footer: {
+    marginTop: theme.spacing(16),
+  },
+  pagination: {
+    margin: theme.spacing(3, 0, 4),
+    display: 'flex',
+    justifyContent: 'space-between',
+  },
+  pageLinkButton: {
+    textTransform: 'none',
+    fontWeight: theme.typography.fontWeightRegular,
```

Comment:
The links definitely look better with `theme.palette.color.primary`. That wasn't an accident.

Reply:
I have tried to minimize the reading distraction of the pagination as much as possible: slim font weight + neutral color.